websites/A/Anizium/presence.ts

```diff
@@ -0,0 +1,114 @@
+interface Video {
+	paused: boolean;
+	duration: number;
+	currentTime: number;
+}
+
+const presence = new Presence({
+		clientId: "1094616557763710976",
+	}),
+	strings = presence.getStrings({
+		viewHome: "general.viewHome",
+	});
```

Comment:
Multi-language strings may not be necessary for this presence.

Reply:
okey i deleted it